examples/seismic/model.py

```diff
@@ -57,21 +57,25 @@ def __init__(self, origin, spacing, vp, nbpml=0, dtype=np.float32,
         self.spacing = spacing
         self.nbpml = nbpml
         self.dtype = dtype
-
+        # Generate a random id with the model.
+        # This is needed for the Gradient test that uses two different models
+        # Model(m) and Model(m0) that uses the same value of m without id
```

Comment:
Ok, I think this here really exposes a severe design flaw that requires an API re-think and a separate PR to fix. I've filed issue #220 to track this.